interface/src/app/utils.js

```diff
@@ -17,4 +17,24 @@ angular.module('adage.utils', [])
 
 // Number of digits in activity value shown on frontend:
 .constant('ActivityDigits', 5)
+
+// Service of global machine learning model, which is shared by multiple
+// modules.
```

Comment:
Oh, I thought it would just be a small change. If we make the service a dependency of `adage.utils` then it will be available everywhere as if it were included right here. Maybe I'm missing something. I will make an issue.